sub-projects/device/src/aws.py

```diff
@@ -0,0 +1,42 @@
+import boto3
+import json
+from PIL import Image
+
+class Rekgoniton(object):
+	"""
+	A class that creates a client object for access to AWS Rekgoniton.
+	"""
+	def __init__(self):
+		self.client = boto3.client("rekognition")
+		self.image_list = ['cameron.jpg','rachel.jpg']
+
+	def compare(self, img):
+		"""
+		Compares a face from the S3 bucket to a passed in image represented in
+		bytes.
+		:param img: target image for comparing
+		"""
+		i = 0
+		while i < len(self.image_list):
```

Comment:
`i` is never changed. This is an infinite while loop

Reply:
So two things here:
1. The `len()` function will get called `len(self.image_list)` amount of times. I would separate it out above the while loop
2. I think a better way to express what is trying to be accomplished here in code is using a `for` loop. Since we are just iterating over each item in the `self.image_list`, and as soon as you find a match just `break`.

Reply:
Oh you mean if there are multiple faces in `img`? Yeah then instead of `break` just build up a list and return something like `True, ["list", "of", "ppl"] if you want